src_assets/common/assets/web/Locale.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+</template>
+
+<script>
+import { createI18n } from "vue-i18n";
+
+// Import translation files
+import de from './locale/de.json'
+import en from './locale/en.json'
+import en_GB from './locale/en-GB.json'
+import en_US from './locale/en-US.json'
+import es from './locale/es.json'
+import fr from './locale/fr.json'
+import it from './locale/it.json'
+import ru from './locale/ru.json'
+import sv from './locale/sv.json'
+import zh from './locale/zh.json'
+
+// Create the i18n instance
+const i18n = createI18n({
+    locale: "en",  // initial locale, todo: how to get this from config?
+    fallbackLocale: "en",  // fallback locale
+    messages: {
+        de: de,
+        en: en,
+        "en-GB": en_GB,
+        "en-US": en_US,
+        es: es,
+        fr: fr,
+        it: it,
+        ru: ru,
+        sv: sv,
+        zh: zh
+    },
+});
+
+export {
+    i18n
+};
+
+export default {
+    created() {
+        this.fetchLocale();
```

Comment:
Todo: determine why this isn't being called

Reply:
Fixed in commit [b4c0a05](https://github.com/LizardByte/Sunshine/pull/2279/commits/b4c0a05b5a77ac8f484647663d239bedb035f246)